app/components/ClientSampleCode.tsx

```diff
@@ -116,5 +117,37 @@ export function ClientSampleCode({
       break
   }
 
-  return <Highlight language={language} code={code} />
+  return (
+    <>
+      <div>
+        {language == 'python' ? (
+          <div>
+            Run this command to install with{' '}
+            <Link href="https://pip.pypa.io/">pip</Link>:
+            <pre>
+              <code className="hljs language-sh">pip install gcn-kafka</code>
+            </pre>
```

Comment:
Replace with a `<Highlight>` element so that it is syntax highlighted and has a copy button.

Reply:
Done